src/utils/common-utils.ts

```diff
@@ -22,3 +23,18 @@ export function flattenObsList(obsList: OpenmrsObs[]): OpenmrsObs[] {
 export function hasRendering(field: OHRIFormField, rendering: RenderType) {
   return field.questionOptions.rendering === rendering;
 }
+
+export function createGalleryEntry(data: AttachmentResponse): Attachment {
+  const attachmentUrl = '/ws/rest/v1/attachment';
+  return {
+    id: data.uuid,
+    src: `${window.openmrsBase}${attachmentUrl}/${data.uuid}/bytes`,
+    title: data.comment,
+    description: '',
+    dateTime: formatDate(new Date(data.dateTime), {
+      mode: 'wide',
+    }),
+    bytesMimeType: data.bytesMimeType,
+    bytesContentFamily: data.bytesContentFamily,
+  };
+}
```

Comment:
Picked this implementation from [esm-patient-attachments-app](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-attachments-app/src/utils.ts), which converts the attachment response into a usable attachment.

Reply:
Yes I understand, within the context of that esm, it makes sense to use the term "gallery" because there is an actual gallery but we don't have one here. Can you revise the naming?

Reply:
Will do.